accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,298 @@
+- Feature Name: Container as a Service Platform integration with SUSE Manager
+- Start Date: 2019-08-05
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC describes how SUSE Manager can be leveraged to perform cross-product integration with Container as a Service Platform.
+
+# Motivation
+[motivation]: #motivation
+
+<!-- - Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Describe the problem you are trying to solve, and its constraints, without coupling them too closely to the solution you have in mind. If this RFC is not accepted, the motivation can be used to develop alternative solutions. -->
+
+SUSE Manager is the systems management tool and must offer to the end-user the possibility to deploy and manage other SUSE products like CaaS Platform.
+
+We will be focusing on the following scenarios:
+
+1. Deploy a CaaS Platform cluster using SUSE Manager
+2. Manage a CaaS Platform cluster using SUSE Manager
+
+
+# Detailed design
+[design]: #detailed-design
+
+<!-- This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the product to understand, and for somebody familiar with the internals to implement.
+
+This section should cover architecture aspects and the rationale behind disruptive technical decisions (when applicable), as well as corner-cases and warnings. Whenever the new feature creates new user interactions, this section should include examples of how the feature will be used. -->
+
+## Deploy a CaaS platform cluster using SUSE Manager
+
+  ### Cluster Deployment
+
+  Under SUSE Manager main menu, a new menu item "CaaS Platform" should be introduced.
+
+  Under that item, it should be possible to create, edit, and remove a cluster.
+
+  To create a cluster, a user has to enter its name.
+
+  Then, the user will be requested to associate a load balancer and a management node to the cluster.
+
+  The cluster is then identified by:
+  - a unique name
+  - the hostnames of the load balancers
+  - the hostname of its management node
+
+  (more on the load balancer and the management node below).
+
+  Every detail about the cluster is saved into corresponding database tables (e.g. `susecaaspcluster`).
+
+  NOTE: the load balancer is unique for the cluster. The management node can manage different clusters at the same time. (TODO: Check with CaaSP team the assumption).
```

Comment:
This is a fair assumption at a first stage. Moreover, I think it would be good to specify that a L4 load balancer should suffice, so it doesn't need to perform TLS termination itself (what would come with more complexity, like handling certificates and CA at LB level).

Reply:
OK, adding level 4 and no TLS termination